test/pipelines/pipelines-it-frameworkcontroller.yml

```diff
@@ -32,6 +32,12 @@ jobs:
       PATH=$HOME/.local/bin:$PATH nnictl package install --name=SMAC
       PATH=$HOME/.local/bin:$PATH nnictl package install --name=BOHB
     displayName: 'Install dependencies for integration tests in frameworkcontroller mode'
+  
+  - script: |
+      cd examples/tuners/customized_tuner
+      python3 setup.py develop
+      nnictl package install ./
```

Comment:
please set PATH like `PATH=$HOME/.local/bin:$PATH nnictl package install --name=SMAC`
and add `set -e` at beginning.

Reply:
Thank you, I will modify them.